proto/felixbackend.pb.go

```diff
@@ -65,21 +65,16 @@
 */
 package proto
 
-import (
-	fmt "fmt"
+import proto1 "github.com/gogo/protobuf/proto"
```

Comment:
I don't think your PR should include any changes in this file.  Can you revert them?